cmake/modules/Buildpmdk.cmake

```diff
@@ -1,4 +1,4 @@
-function(build_pmem)
+function(build_pmdk)
```

Comment:
might want to correct the typo in the title of the commit message.

Reply:
@CongMinYin please reconsider this comment. i believe [serious](https://www.merriam-webster.com/dictionary/serious) is a typo.

Reply:
@tchaikov sorry, revised. Forget to force push after modification.